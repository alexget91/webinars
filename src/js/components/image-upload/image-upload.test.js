import React from "react";
import renderer from "react-test-renderer";
import ImageUpload from "./image-upload";

it(`Image upload correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<ImageUpload
      onImageUpload={jest.fn()}
      cssClass={`cssClass`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
