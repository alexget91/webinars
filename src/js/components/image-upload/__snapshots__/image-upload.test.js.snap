// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image upload correctly renders after relaunch 1`] = `
<section
  className="image-upload cssClass "
>
  <aside
    className="image-upload__preview-container"
  >
    <button
      className="image-upload__clear"
      onClick={[Function]}
      title="Clear"
    />
  </aside>
  <div
    className="image-upload__drop-zone"
    onBlur={[Function]}
    onClick={[Function]}
    onDragEnter={[Function]}
    onDragLeave={[Function]}
    onDragOver={[Function]}
    onDrop={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    tabIndex={0}
  >
    <input
      accept="image/jpeg, image/png"
      autoComplete="off"
      multiple={false}
      onChange={[Function]}
      onClick={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
      tabIndex={-1}
      type="file"
    />
    <div
      className="image-upload__text-container"
    >
      <p
        className="image-upload__text"
      >
        select an image file to upload or drag it here
      </p>
    </div>
  </div>
</section>
`;
